packages/components/addon/components/hds/app-footer/legal-links.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+export default class HdsAppFooterLegalLinksComponent extends Component {
+  /**
+   * @param hrefForSupport
+   * @type {string}
+   * @description The href value of the "Support" link
+   */
+  get hrefForSupport() {
+    return this.args.hrefForSupport ?? 'https://www.hashicorp.com/support';
+  }
+
+  /**
+   * @param hrefForTerms
+   * @type {string}
+   * @description The href value of the "Terms" link
+   */
+  get hrefForTerms() {
+    return (
+      this.args.hrefForTerms ?? 'https://www.hashicorp.com/terms-of-service'
```

Comment:
This is the only default value of these LegalLinks that I'm curious about as this drives to the `.com` _website_ terms & conditions. Checking the HCP terms link; it drives to an internal page (/terms-of-service) that is specific to HCP, so this is probably the best solution as it will likely be overridden.

Reply:
@didoo Any opinion?